dashboard/src/components/OperatorView/OperatorView.test.tsx

```diff
@@ -40,32 +71,20 @@ it("shows an error if the operator doesn't have any channel defined", () => {
 });
 
 it("renders a full OperatorView", () => {
+  const wrapper = shallow(<OperatorView {...defaultProps} operator={defaultOperator} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it("selects the default channel", () => {
   const operator = {
-    metadata: {
-      name: "foo",
-      namespace: "kubeapps",
-    },
+    ...defaultOperator,
     status: {
-      provider: {
-        name: "Kubeapps",
-      },
-      channels: [
-        {
-          currentCSVDesc: {
-            displayName: "Foo",
-            version: "1.0.0",
-            description: "this is a testing operator",
-            annotations: {
-              capabilities: "Basic Install",
-              repository: "github.com/kubeapps/kubeapps",
-              containerImage: "kubeapps/kubeapps",
-              createdAt: "one day",
-            },
-          },
-        },
-      ],
+      ...defaultOperator.status,
+      channels: [{ name: "alpha" }, defaultOperator.status.channels[0]],
     },
   };
   const wrapper = shallow(<OperatorView {...defaultProps} operator={operator as any} />);
-  expect(wrapper).toMatchSnapshot();
+  expect(wrapper.find(OperatorDescription).prop("description")).toEqual(
+    "this is a testing operator",
+  );
```

Comment:
Shouldn't something here be checking the channel? (ie. that the default channel is still beta? Or is the test name inaccurate?)

Reply:
it's indirectly checking the channel, here the default channel is `beta` even though it's not the first one. Since it's getting the right description (the one for the beta channel), it's choosing the right channel.